website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -40,7 +40,7 @@ description: >-
 
 [Policy Enforcement](/terraform/cloud-docs/policy-enforcement) lets you use the policy-as-code frameworks Sentinel and Open Policy Agent (OPA) to apply policy checks to Terraform Cloud workspaces.
 
-Policy sets let you group related policies by attributes like environment or region. You can then apply each policy set to one or more workspaces. For these workspaces, Terraform Cloud checks every Terraform plan against each policy in the set.
+Policy sets let you group related policies by attributes like environment or region. You can then apply each policy set to one or more workspaces and projects. For these workspaces, Terraform Cloud checks every Terraform plan against each policy in the set.
```

Comment:
should "these" be here?

Reply:
I didn't change that text so that exists in production today so I'm going to leave it.